[dropbear:280012:5 autogenerated test from ./results/dropbear.results]

log 1 pass = Mar 24 15:25:51 buffalo1 dropbear[4092]: bad password attempt for 


alert = 5
decoder = dropbear
rule = 280012

[dropbear:280014:2 autogenerated test from ./results/dropbear.results]

log 1 pass = Jan 22 21:21:20 buffalo1 dropbear[3859]: exit before auth: Failed to get kex value
log 2 pass = Mar 22 16:29:14 buffalo1 dropbear[25121]: exit before auth: Exited normally
log 3 pass = Mar 22 16:29:51 buffalo1 dropbear[25132]: exit before auth: Incompatible remote version 
log 4 pass = Mar 22 16:29:53 buffalo1 dropbear[25139]: exit before auth: Incompatible remote version 


alert = 2
decoder = dropbear
rule = 280014

[dropbear:280011:1 autogenerated test from ./results/dropbear.results]

log 1 pass = Jan 22 21:21:20 buffalo1 dropbear[3859]: Premature kexdh_init message received


alert = 1
decoder = dropbear
rule = 280011

[dropbear:280016:1 autogenerated test from ./results/dropbear.results]

log 1 pass = Mar 22 16:22:17 buffalo1 dropbear[25017]: password auth succeeded for 


alert = 1
decoder = dropbear
rule = 280016

[dropbear:280013:1 autogenerated test from ./results/dropbear.results]

log 1 pass = Mar 22 16:22:18 buffalo1 dropbear[25017]: exit after auth (root): Disconnect received


alert = 1
decoder = dropbear
rule = 280013

